.gitlab-ci.yml

```diff
@@ -1597,13 +1598,33 @@ twistlock_scan:
     - TWISTLOCK_PASS=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.twistlock_password --with-decryption --query "Parameter.Value" --out text)
     - scan () { echo -e "\n==== Scanning $1 ====\n"; docker pull $1 > /dev/null; /twistcli images scan --address="$TWISTLOCK_URL" --user="$TWISTLOCK_USER" --password="$TWISTLOCK_PASS" --vulnerability-threshold=$THRESHOLD --details $1; }
   script:
-    - scan ${SRC_AGENT}:${SRC_TAG}-py2
-    - scan ${SRC_AGENT}:${SRC_TAG}-py3
-    - scan ${SRC_AGENT}:${SRC_TAG}-py2-jmx
-    - scan ${SRC_AGENT}:${SRC_TAG}-py3-jmx
+    - scan ${SRC_AGENT}:${SRC_TAG}-6-py2
+    - scan ${SRC_AGENT}:${SRC_TAG}-6-py3
+    - scan ${SRC_AGENT}:${SRC_TAG}-6-py2-jmx
+    - scan ${SRC_AGENT}:${SRC_TAG}-6-py3-jmx
```

Comment:
Internal Agent 6 images do not have (yet) the `-6` part, contrary to the Agent 7 images. That will be fixed in #4336.

Reply:
Fixed 👍